build.gradle

```diff
@@ -1,7 +1,7 @@
 ext {
     wordPressUtilsVersion = '2.4.0'
     wordPressLoginVersion = '0.12.0'
-    gutenbergMobileVersion = 'v1.72.0-alpha5'
+    gutenbergMobileVersion = '4668-2095f19d1a7ed6d52b0f394f9579e66bb4876f0c'
```

Comment:
Same as https://github.com/wordpress-mobile/WordPress-iOS/pull/18158#discussion_r829904208.

Reply:
I replied over at the WordPress-iOS repo. `2095f19d1a7ed6d52b0f394f9579e66bb4876f0c` is ahead of the bundle steps and has the latest changes aside from the release note updates. But those will be included once we tag the release in Gutenberg Mobile and update `gutenbergMobileVersion` to point that tag.

Reply:
Looks like we have a new bundle update at this point, let's point to the latest. Where I'm coming from: I prefer to start the merge train only after the whole PR chain is up-to-date and green. WDYT?